src/server/modules/user/services/UrlManagementService.ts

```diff
@@ -128,18 +134,22 @@ export class UrlManagementService implements interfaces.UrlManagementService {
     }
 
     // Success
-    const result = await this.urlRepository.update(url, {
+    return this.urlRepository.update(url, {
       userId: newUserId,
     })
-
-    return result
   }
 
   getUrlsWithConditions: (
     conditions: UserUrlsQueryConditions,
   ) => Promise<UrlsPaginated> = (conditions) => {
     return this.userRepository.findUrlsForUser(conditions)
   }
+
+  getTagsWithConditions: (
+    conditions: UserTagsQueryConditions,
+  ) => Promise<string[]> = (conditions) => {
+    return this.tagRepository.findTagsWithConditions(conditions)
+  }
```

Comment:
Feels unintuitive that `getTagsWithConditions` is in the `UrlManagementService` when it doesn't actually interact with the `urlRepository` or anything. Also, the only reason we imported `tagRepository` into the service is for this function. What do you think of creating a separate `TagManagementService` for this function? Then future extensions to tagging can also belong to that service?

Reply:
I discussed with @halfwhole on this, we were considering the option to create the tagmanagementservice too, but this will be the only function in the tagmanagement service and tags belong to url as well. So I think this should be fine. We can consider refactoring if there are more operations on tags

Reply:
done